lib/Utils.js

```diff
@@ -192,8 +192,9 @@ Utils.Endpoints = {
   orgs: (schemaVersion, orgId) => `${Utils.Endpoints.version(schemaVersion)}/organizations${Utils.Endpoints.getIdPartFromId(orgId)}`,
   servicesByDomain: (domainType, domainId, serviceId, schemaVersion) => `${Utils.Endpoints.version(schemaVersion)}/${domainType}/${domainId}/data-links`,
```

Comment:
There is still data-links in the URL here. Is this something that is no longer used or is it "data-links" in the new API?

Reply:
Yes, removed.